### PR TITLE
Make these styles available once and for all to all the people

### PR DESCRIPTION
**Pull request by jmelvnsn**
@theonion/front-end Long time comin.

### CODE REVIEW

**Comment by kand** [comment]
I like it.

**Comment by spra85** [comment]
LGTM.  Do we feel like we should take a look at test links before pushing to each site?

**Comment by jmelvnsn** [comment]
@spra85 @kand We need to do a full vetting of each site before installing. Will need to remove all campaign-display declarations in local styles (or at least the bits that are duplicates of this PR). I created the PR bc I couldn't stand to write them again while working on the ClickHole video experience. I can make some tickets for this work if anyone wants to pick it up and run with it.

**Comment by jmelvnsn** [comment]
https://trello.com/c/J4DY0smh/1123-make-campaign-displat-styles-globally-available-and-remove-local-declarations-that-are-duplicates-of-the-bulbs-elements-dist-sty

File: elements/campaign-display/campaign-display.scss [hunk 1]
```diff
@@ -1,8 +1,24 @@
 campaign-display {
-  display: block;
-
+  color: rgba(112, 112, 112, 1);
+  display: inline-block;
+  font-family: 'Helvetica', Arial, sans-serif;
+  font-size: 10px;
+  text-transform: uppercase;
+  .inner a {
```
Comment (kand) [inline elements/campaign-display/campaign-display.scss]: Want to note, `.inner` seems like too generic a class name, something else might style this on accident. But that change is much bigger than this PR and includes `omni`.

File: elements/campaign-display/campaign-display.scss [hunk 2]
```diff
@@ -15,7 +15,6 @@ campaign-display {
     }
   }
   .campaign-display-logo {
-    margin-left: 15px;
```
Comment (jmelvnsn) [inline elements/campaign-display/campaign-display.scss]: revert this please

**Comment by kand** [comment]
@MelizzaP @jmelvnsn can we merge this PR?

**Comment by MelizzaP** [comment]
@kand no, still need to update clickhole and avclub

File: elements/campaign-display/campaign-display.scss (hunk 1 above)
Comment (brycedorn) [inline elements/campaign-display/campaign-display.scss]: [confirmed](http://www.clickhole.com/search?q=sponsored+by), CH is picking up `.inner` twice:

screen shot 2017-02-27 at 11 05 41 am

**Comment by collin** [comment]
# @jmelvnsn

**Pull request merged**